### ISSUE TITLE
Update to React 15.5

### ISSUE THREAD

**Issue by dlebedynskyi** [Toxblh/react-hoc-outside-click#1]
React 15.5 is giving warning on 
```
warning.js:36 Warning: Accessing PropTypes via the main React package is deprecated. Use the prop-types package from npm instead.
```
Can we get an update? 
https://facebook.github.io/react/blog/2017/04/07/react-v15.5.0.html#migrating-from-react.proptypes

**Comment by 3ptmonster** [comment]
#2 Implemented 'prop-types' module per React 15.5

**Comment by Toxblh** [comment]
Thanks @3ptmonster
